AdyenActions/Components/3DS2/Action handlers/ThreeDS2CompactActionHandler.swift

```diff
@@ -47,11 +47,13 @@ internal final class ThreeDS2CompactActionHandler: AnyThreeDS2ActionHandler, Com
                   service: AnyADYService = ADYServiceAdapter(),
                   appearanceConfiguration: ADYAppearanceConfiguration = ADYAppearanceConfiguration(),
                   coreActionHandler: AnyThreeDS2CoreActionHandler? = nil,
-                  delegatedAuthenticationConfiguration: ThreeDS2Component.Configuration.DelegatedAuthentication? = nil) {
+                  delegatedAuthenticationConfiguration: ThreeDS2Component.Configuration.DelegatedAuthentication? = nil,
+                  presentationDelegate: PresentationDelegate?) {
```

Comment:
Maybe add `coreActionHandler`, `delegatedAuthenticationConfiguration`, `presentationDelegate` to meta-docs?